ros2/naive/include/naive/incrementer.hpp

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
```

Comment:
I'm guessing ament is going to want a copyright here

Reply:
I added the copyright, I think it's the right one?